src/content/docs/tips/biome.mdx

```diff
@@ -0,0 +1,138 @@
+</Aside>
+
+<Steps>
+
+1. Install biome using your preferred package manager:
+    <Tabs syncKey="biome">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+        npm install --save-dev --save-exact @biomejs/biome
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpm add --save-dev --save-exact @biomejs/biome
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        yarn add --dev --exact @biomejs/biome
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bun add --dev --exact @biomejs/biome
+        ```
+      </TabItem>
+    </Tabs>
+
+2. Next create the biome configuration:
+
+    <Tabs syncKey="biome">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+        npx @biomejs/biome init
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpm biome init
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        yarn biome init
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bunx biome init
+        ```
+      </TabItem>
+      <TabItem label="deno" icon="deno">
+        ```sh
+        deno run -A npm:@biomejs/biome init
+        ```
+      </TabItem>
+    </Tabs>
+
+3. Provide the overrides property inside biome config to support `.astro` files:
+
+    ```js
+    {
+      "overrides": [
+        {
+          "include": ["*.astro"],
+          "linter": {
+            "rules": {
+              "style": {
+                "useConst": "off",
+                "useImportType": "off"
+              }
+            }
+           }
+        }
+      ]
+    }
+    ```
+
+4. Run Biome's CLI to lint & format your project's files:
+  
+    <Tabs syncKey="biome">
+        <TabItem label="npm" icon="seti:npm">
+          ```sh
+          npx @biomejs/biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="pnpm" icon="pnpm">
+          ```sh
+          pnpm biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="yarn" icon="seti:yarn">
+          ```sh
+          yarn biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="bun" icon="bun">
+          ```sh
+          bunx biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="deno" icon="deno">
+          ```sh
+          deno run -A npm:@biomejs/biome check --write .
+          ```
+        </TabItem>
+      </Tabs>
+</Steps>
+
+<Aside>
+In case of an existing configuration for ESLint and Prettier, you might want to read Biome's guide for [Migrate from ESLint and Prettier](<https://biomejs.dev/guides/migrate-eslint-prettier/>)
+</Aside>
+
+
+## Adding an npm script
```

Comment:
Maybe we don't want to refer to `npm` here. Would `package.json` script work, or do we have any other ideas?

Reply:
fair point, more inclusive much better ✔️ 
```suggestion
## Adding an package.json script
```